app/lib/project/OTFExportController.js

```diff
@@ -0,0 +1,95 @@
+define([
+    'metapolator/errors'
+  , 'metapolator/rendering/glyphBasics'
+  , 'metapolator/rendering/OpenTypePen'
+  , 'ufojs/tools/pens/PointToSegmentPen'
+  , 'opentype'
+  , 'metapolator/models/MOM/Glyph'
+  , 'metapolator/timer'
+], function(
+    errors
+  , glyphBasics
+  , OpenTypePen
+  , PointToSegmentPen
+  , opentype
+  , MOMGlyph
+  , timer
+) {
+    "use strict";
+
+    function OTFExportController(master, model, masterName) {
+        this._master = master;
+        this._model = model;
+        this._masterName = masterName;
+    }
+    var _p = OTFExportController.prototype;
+
+    _p.do_export = function() {
+        var glyphs = this._master.children
+          , glyph
+          , updatedUFOData
+          , i, l, v, ki, kil, k, keys
+          , style
+          , time, one, total = 0
+          , font
+          , otf_glyphs = []
+          , renderer = {
+                  penstroke: glyphBasics.renderPenstrokeOutline
+                , contour: glyphBasics.renderContour
+            }
+          ;
+
+        console.warn('exporting OTF ...');
+        for(i = 0,l=glyphs.length;i<l;i++) {
+            var otPen = new OpenTypePen()
+              , pen = new PointToSegmentPen(otPen)
+              ;
+
+            glyph = glyphs[i];
+            style = this._model.getComputedStyle(glyph);
+            time = timer.now();
+
+            // Allow the glyph ufo data to be updated by the CPS.
```

Comment:
As this block (line 37 to 50) is now perfectly duplicated from UFOExportController, I would consider it ripe to become a method of it's own in MOM/Glyph.

We can do this after the merge, but this would be a big maintenance deal.